lib/keila_web/live/segment_edit_live.ex

```diff
@@ -2,30 +2,34 @@ defmodule KeilaWeb.SegmentEditLive do
   use KeilaWeb, :live_view
   alias Keila.Contacts
 
-  @fields %{
-    "inserted_at" => %{type: "date", label: gettext("Date added")},
-    "email" => %{type: "string", label: gettext("Email")},
-    "first_name" => %{type: "string", label: gettext("First name")},
-    "last_name" => %{type: "string", label: gettext("Last name")},
-    "double_opt_in_at" => %{type: "date", label: gettext("Double opt-in date")},
-    "data" => %{type: "custom", label: gettext("Custom data")}
-  }
+  defp fields do
```

Comment:
Very good catch that this change was necessary to avoid gettext running at compile-time!

Reply:
I'm learning those kinds of Elixir behaviors while doing. 
I had to scratch my head to understand why I didn't see french translation in this part of the application 🤯